### ISSUE TITLE
Sync auto fails with add / remove changes

### ISSUE THREAD

**Issue by ryanking1809** [alloc/wana#26]
I'm trying to get a getter to update when adding / removing object properties.  

The example below acts a little strange. The getter updates when I add the first property to the object, but never afterwards.

https://codesandbox.io/s/wana-object-getters-u1q8b

```js
const lookup = o({ 1: {}, 2: {}, 3: {} });
const list = o(() => Object.values(lookup));
auto(() => console.log("auto", { ...lookup }, [...list()]), { sync: true });

// auto fires here
lookup["4"] = {};

// auto doesn't fire
lookup["5"] = {};
// auto doesn't fire
lookup["6"] = {};

// auto doesn't fire
delete lookup["6"];
```

---

EDIT:  Can confirm this is only an issue when `sync: true`, the following works as expected.
```js
const lookup = o({ 1: {}, 2: {}, 3: {} });
const list = o(() => Object.values(lookup));
auto(() => console.log("auto", { ...lookup }, [...list()]));

lookup["4"] = {};
setTimeout(() => (lookup["5"] = {}), 500);
setTimeout(() => (lookup["6"] = {}), 500);
setTimeout(() => delete lookup["6"], 500);
```

**Comment by aleclarson** [comment]
🎉 Fixed by 128aa16df37810227f26b1d0ab19f4f7e780ec10 in `v0.13.4`

**Comment by ryanking1809** [comment]
awesome thanks!
